mirroroperator/registrymirror.py

```diff
@@ -487,6 +502,26 @@ def generate_stateful_set(self):
                                 args=["-c", script],
                                 volume_mounts=generate_ca_certs_volume_mounts,
                                 resources=resources,
+                            ),
+                            client.V1Container(
+                                name="get-nameservers",
+                                image="busybox",
+                                command=["/bin/sh"],
+                                args=["-c", script_munge_nameservers],
+                                volume_mounts=[
+                                    client.V1VolumeMount(
+                                        name="nginx-config",
+                                        mount_path="/var/lib/conf.d",
```

Comment:
why `/var/lib/conf.d` and not `/etc/nginx/conf.d`?

Reply:
as above, I'm not sure I understand the need for this extra container

Reply:
i feel that `/var/lib/conf.d` confuses things too much and obfuscates what is in that directory

Reply:
this still references `/var/lib/conf.d`

Reply:
oops.....missed this. Will fix. thanks